contracts/coordinator/test/mixins.ts

```diff
@@ -366,25 +371,24 @@ describe('Mixins tests', () => {
                     transactionSignerAddress,
                     approvalExpirationTimeSeconds,
                 );
-                expectContractCallFailedAsync(
-                    mixins.assertValidCoordinatorApprovals.callAsync(
-                        transaction,
-                        transactionSignerAddress,
-                        transaction.signature,
-                        [approvalExpirationTimeSeconds],
-                        [approval.signature],
-                        { from: approvalSignerAddress2 },
-                    ),
-                    RevertReason.InvalidOrigin,
+
+                const tx = mixins.assertValidCoordinatorApprovals.callAsync(
+                    transaction,
+                    transactionSignerAddress,
+                    transaction.signature,
+                    [approvalExpirationTimeSeconds],
+                    [approval.signature],
+                    { from: approvalSignerAddress2 },
                 );
+                expect(tx).to.revertWith(new CoordinatorRevertErrors.InvalidOriginError(transactionSignerAddress));
             });
         }
     });
     describe('Batch order approvals', () => {
         for (const fnName of [
             ...exchangeConstants.BATCH_FILL_FN_NAMES,
             ...exchangeConstants.MARKET_FILL_FN_NAMES,
-            ExchangeFunctionName.MatchOrders,
+            ...exchangeConstants.MATCH_ORDER_FN_NAMES,
```

Comment:
Nah `MixinCoordinatorApprovalVerifier` doesn't support the batch match functions currently. The match order functions are here because `exchangeDataEncoder` handles them similarly to the batch fill functions.